styles.css

```diff
@@ -0,0 +1,253 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    background: url(assets/background1.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    font-family: Serif;
```

Comment:
lowercase s

Reply:
This is done, just a note that Php storm default value uses capital S.